tests/core/test_polydata.py

```diff
@@ -1107,3 +1107,34 @@ def test_regular_faces_mutable():
     mesh = pv.PolyData.from_regular_faces(points, faces)
     mesh.regular_faces[0, 2] = 3
     assert np.array_equal(mesh.faces, [3, 0, 1, 3])
+
+
+@pytest.mark.parametrize(
+    "attribute,value",
+    [
+        ("faces", [3, 1, 2, 3, 3, 0, 1]),
+        ("strips", np.array([5, 4, 3, 2, 0])),
+        ("lines", [4, 0, 1, 2, 2, 3, 4]),
+        ("verts", [1, 0, 1]),
+    ],
+)
+def test_init_with_invalid_padding(arr):
+    """This test checks that an error is raised when the padding is invalid.
+
+    More precisely, if a PointSet is created with ``verts``, ``lines``, ``faces``
+    or ``strips`` we are expecting a ValueError to be raised if the array length
+    is not compatible with the padding structure.
+
+    """
+
```

Comment:
The function name is sufficient to read the contents, so let's remove the docstring as you would any other test function.
```suggestion
```